pom.xml

```diff
@@ -69,27 +69,27 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-step-api</artifactId>
-            <version>2.7</version>
+            <version>2.13</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.8</version>
+            <version>2.23.1</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-support</artifactId>
-            <version>2.12</version>
+            <version>2.14</version>
```

Comment:
🐛 Why not 2.16?  It has the stop implementation, etc etc

Reply:
2.14 is the last one that doesn't depend on core 2.60.2: https://github.com/jenkinsci/workflow-support-plugin/commit/0fa02e3807868ba3bd38526e9e13df9510d6dc89